xonsh/history/base.py

```diff
@@ -103,8 +103,8 @@ def append(self, cmd):
         Parameters
         ----------
         cmd: dict
-            A dict contains informations of a command. It should contains
-            the following keys like ``inp``, ``rtn``, ``ts`` etc.
+            A dict contains information on a command. It should contain
+            the following keys, like ``inp``, ``rtn``, ``ts`` &etc.
```

Comment:
I don't actually know any of the details about the history setup here, so I'm not sure what's optional and what's required.  And the sentence currently doesn't actually describe what any of those things _mean_.

Maybe:

"This dict contains information about a command.  It should contain keys `'inp'`, `'rtn'`, `'out'`, and `'ts'`, which have the same forms and meanings as the corresponding instance variables of the `HistoryEntry` class."

It's a pretty ugly sentence, but it is a bit more descriptive.